commit ec12f7cd871f4fad8af269c82213964a151f5d36 (refs/changes/27/4227/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-06-15T19:05:48+00:00 (1 year, 4 months ago)
    
    MANTA-3719 Need Firewall Rules for Postgres
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/boot/setup.sh b/boot/setup.sh
index 49b403a..c892924 100755
--- a/boot/setup.sh
+++ b/boot/setup.sh
@@ -77,6 +77,9 @@ function common_enable_services {
 
     svccfg import /opt/smartdc/manatee/smf/manifests/sitter.xml
     # For SDC we'll let configure decide if it wants to enable sitter or not.
+
+    echo "Starting waferlock"
+    svccfg import /opt/smartdc/waferlock/smf/manifests/waferlock.xml
 }
 
 function common_manatee_setup {
@@ -175,7 +178,7 @@ function add_manatee_profile_functions {
 
 
 # Local manifests
-CONFIG_AGENT_LOCAL_MANIFESTS_DIRS=/opt/smartdc/$role
+CONFIG_AGENT_LOCAL_MANIFESTS_DIRS="/opt/smartdc/$role /opt/smartdc/waferlock"
 
 # Include common utility functions (then run the boilerplate)
 source /opt/smartdc/boot/lib/util.sh
@@ -189,6 +192,7 @@ add_manatee_profile_functions
 sdc_log_rotation_add manatee-sitter /var/svc/log/*manatee-sitter*.log 1g
 sdc_log_rotation_add manatee-snapshotter /var/svc/log/*manatee-snapshotter*.log 1g
 sdc_log_rotation_add manatee-backupserver /var/svc/log/*manatee-backupserver*.log 1g
+sdc_log_rotation_add waferlock /var/svc/log/*waferlock*.log 1g
 sdc_log_rotation_add manatee-postgres /var/pg/postgresql.log 1g
 sdc_log_rotation_setup_end
 
